### PR TITLE
Merge v1.0 code base

### PR DESCRIPTION
**Pull request by haydenyoung**
Merging the latest OrbitDB development from version 1.0.0 code base into the main OrbitDB repo.

### CODE REVIEW

File: docs/ACCESS_CONTROLLERS.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+# Access Controllers
```
Comment (aphelionz) [inline docs/ACCESS_CONTROLLERS.md:1]: Should we link to this from the main README.md?

File: docs/CONNECTING_PEERS.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+# Connecting Peers
```
Comment (aphelionz) [inline docs/CONNECTING_PEERS.md:1]: Likewise, should we link to this from the main README.md?

File: docs/ACCESS_CONTROLLERS.md (hunk 1 above)
Comment (haadcode) [reply docs/ACCESS_CONTROLLERS.md:1]: Yup, makes sense. We'll iterate on the main README in the upcoming PRs 👍

**Review by haadcode — APPROVED** [review]
With this, v1.0 is *finally* upon us! 🎉 

Thank you @haydenyoung for all your hard work in making this happen 🙏 ❤️

**Pull request merged**